linkml-schema/kbmodel.yaml

```diff
@@ -96,34 +78,222 @@ classes:
         inlined_as_list: true
         range: genome assembly
 
+  process output:
+    description: >-
+      The output of a process.
+    slots:
+      - label
+    attributes:
+      created by process id:
+        description: >-
+          The id of the process that created this object.
+      created by process type:
+        description: >-
+          The type of process that created this object.
+
+  # 028 - donors.ipynb
+  donor:
+    is_a: entity
+    mixins:
+      - thing with taxon
+    description: >-
+      A person or organism that is the source of a biological sample.
+    slots:
+      - label
+      - sex
+      - date of birth
+      - date of death
+      - age at death
+      - full genotype
+
+
+  # 027 - brain_extraction.ipynb
+  brain segment:
+    is_a: process output
+    slots:
+      - barcode
+    attributes:
+      anatomical division:
+    
+
+  # 026 - brain_segment_sectioning.ipynb
+  brain section:
+    is_a: process output
+    slots:
+      - barcode
+    attributes:
+      ordinal:
+        description: >-
+          The ordinal number of the section.
+        range: integer
+
+
+  # 025b - tissue_dissectioning.ipynb
+  tissue sample:
+    is_a: process output
+    attributes:
+      roi plan:
+      region of interest label:
+        description: >-
+          The label of the region of interest.
+
+
+  # 024 - cell_dissociation.ipynb
+  dissociated cell sample:
+    is_a: process output
+    attributes:
+      cell prep type:
+      facs population plan: 
+
+
+  #TODO:
+  enriched cell sample:
+
+
+  # 023 - cell_barcoding.ipynb
+  barcoded cell sample:
+    is_a: process output
+    attributes:
+      port well:
+      sample quallity count:
+
+
+  # 022 - cnda_amplification.ipynb
+  amplified cdna:
+    is_a: process output
+    attributes:
+      amplified quantity:
+        description: >-
+          Amount of cDNA amplified (ng).
+        range: float
+      pcr cycles:
+      percent cdna longer than 400bp:
+      rna amplification pass fail:
+    slots:
+      - method
+
+
+  # 021 - library_constructions.ipynb
+  library:
+    is_a: process output
+    attributes:
```

Comment:
there is also `library_input_ng` in the notebook

Reply:
`library_input_ng` is equivalent to `input amount` in the model